tools/MDTraj/trajectory_converter.xml

```diff
@@ -0,0 +1,61 @@
+<tool id="mdtraj_traj_converter" name="Trajectory Converter" version="1.9.1">
+    <description>convert gromacs trr and gro formats to dcd and pdb formats</description>
+    <requirements>
+        <requirement type="package" version="1.9.1">mdtraj</requirement>
+        <requirement type="package" version="2018.2">gromacs</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+      ln -s '$gro_input' ./groinp.gro &&
+      ln -s '$trr_input' ./trrinp.trr &&
+      mdconvert -o ./dcdout.dcd ./trrinp.trr &&
+      cp ./dcdout.dcd '$dcdout' &&
+      gmx editconf  -f ./groinp.gro -o ./pdbout.pdb &&
```

Comment:
you can replace pdbout.pdb here with '$pdbout' and save the copy in the next line